mods/d2k/rules/infantry.yaml

```diff
@@ -227,3 +227,22 @@ saboteur:
 	Voiced:
 		VoiceSet: SaboteurVoice
 
+nsfremen:
+	Inherits: fremen
+	Valued:
+		Cost: 0
+	Tooltip:
+		Name: Fremen
+		Description: Elite sniper infantry unit\n  Strong vs Infantry\n  Weak vs Vehicles
+	Buildable:
+		Queue: Infantry
+		BuildPaletteOrder: 105
+		Prerequisites: ~disabled
+	RevealsShroud:
+		Range: 4c768
+	Mobile:
+		Speed: 66	##33
```

Comment:
Yes. He doubled the speed over the result of converting the original values to OpenRA.

Actor speed will be dealt with in a separate PR, so I didn't bother changing this as it's not a buildable actor anyway.